README.md

```diff
@@ -1,109 +1,101 @@
-<span align="center"> 
+<span align="center">
+ # Checkpoint
```

Comment:
@salma77 This was also broken in that image. I can see the text as it is `#header` and its not formatted as a heading